packages/node_modules/@ciscospark/react-component-typing-avatar/src/index.js

```diff
@@ -16,6 +16,11 @@ function TypingAvatar(props) {
   }
   return (
     <div className={classNames(`ciscospark-typing-avatar`, styles.typingAvatar)}>
+      <div className={classNames(`ciscospark-typing-avatar-tooltip`, styles.tooltip)}>
+        <div className={classNames(`ciscospark-typing-avatar-tooltip-text`, styles.tooltipText)}>
+          {name}
+        </div>
+      </div>
```

Comment:
Should this be it's own component so we can use it in other places?

Reply:
I'm going to be implementing it in the group icon as well and try to figure out how to make it its own component. Next PR :)